frontend/public/reducers/ui.ts

```diff
@@ -30,7 +30,7 @@ export function getDefaultPerspective() {
       activePerspective = defaultPerspective.properties.id;
     }
   }
-  return activePerspective;
+  return activePerspective || undefined;
```

Comment:
This is to coerce the expectation (`undefined` means no perspective) vs. underlying storage implementation (`null` means no such item in storage).

Reply:
> Is something breaking without this?
> I'd rather not distinguish between null and undefined here.

Without this, `__tests__/reducers/ui.spec.ts` fails on

```ts
expect(getDefaultPerspective()).toBeUndefined();
```

(expected `undefined` but received `null`)

Reply:
A bad test. `null` and `undefined` should be equivalent for this assertion.